### PR TITLE
Run jshint in Travis.

### PR DESCRIPTION
**Pull request by mythmon**
r? (pending Travis passing)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+sudo: false
```
Comment (mythmon) [inline .travis.yml]: This allows us to use the new docker-based Travis infrastructure, which is faster and allows us to use caching in the future.

**Comment by mythmon** [comment]
I'm not sure that the changes I made to fix the jshint errors follow the right style, but it does make jshint happy.

**Comment by mythmon** [comment]
I updated the travis notification settings. Now it doesn't email, and spams irc. This is how we have it set on Kitsune. I'm not sure if that is what you want here.

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
 language: node_js
 node_js:
-   - "0.10"
+- "0.10"
+script: jshint app scripts
+notifications:
+  email: false
```
Comment (Rik) [inline .travis.yml]: Actually, I like emails :)
```yml
email:
  recipients:
    - anthony@mozilla.com
    - atopal@mozilla.com
    - rdalal@mozilla.com
  on_success: change
  on_failure: always
```

**Comment by Rik** [comment]
r+

**Comment by mythmon** [comment]
9a113c2 Fix jshint errors.
e2d9351 Run jshint in Travis

**Pull request closed without merging**